cmd/deploy.go

```diff
@@ -333,3 +435,39 @@ func getDeployResultTable(deploys []client.DeployResult) (string, []string) {
 	header := "\x1b[00mSTATUS\x1b[0m\tCLUSTER\tENVIRONMENT\tAPPLICATION\tVERSION\tDEPLOY_ID\tMESSAGE"
 	return header, rows
 }
+
+func getAPIClient(auroraConfig, overrideToken, overrideCluster string) (*client.ApiClient, error) {
+	api := DefaultApiClient
+	api.Affiliation = auroraConfig
+
+	if overrideCluster != "" && !AO.Localhost {
+		c := AO.Clusters[overrideCluster]
+		if !c.Reachable {
+			return nil, errors.Errorf("%s cluster is not reachable", overrideCluster)
+		}
+
+		api.Host = c.BooberUrl
+		api.Token = c.Token
+		if overrideToken != "" {
+			api.Token = overrideToken
+		}
+	}
+
+	return api, nil
+}
+
+func getDeployClient(unit *deploymentUnit) client.DeployClient {
+	var updateClient *client.ApiClient
+	if AO.Localhost {
+		updateClient = DefaultApiClient
+		updateClient.Affiliation = unit.auroraConfig
+	} else {
+		token := unit.cluster.Token
+		if unit.overrideToken != "" {
+			token = unit.overrideToken
+		}
+		updateClient = client.NewApiClient(unit.cluster.BooberUrl, token, unit.auroraConfig)
```

Comment:
Må ta med refName for å kunne deploye fra en annen branch i AuroraConfig.
RefName finnes i AO